src/Features/CSharp/Portable/Structure/Providers/SwitchStatementStructureProvider.cs

```diff
@@ -11,17 +12,31 @@ namespace Microsoft.CodeAnalysis.CSharp.Structure
 {
     internal class SwitchStatementStructureProvider : AbstractSyntaxNodeStructureProvider<SwitchStatementSyntax>
     {
-        protected override void CollectBlockSpans(
-            SwitchStatementSyntax node,
-            ArrayBuilder<BlockSpan> spans,
-            OptionSet options,
-            CancellationToken cancellationToken)
+        private readonly bool includeInternalStructures;
+      internal  SwitchStatementStructureProvider(bool includeInternalStructures) : base()
+        {
+            this.includeInternalStructures = includeInternalStructures;
+        }
```

Comment:
1. whitespace and indentation issue.
2. no need for `: base()`
3. remove `includeInternalStructures`

Reply:
`includeInternalStructures` is pending a design review.